ha/core/cluster/cluster_manager.py

```diff
@@ -267,12 +267,13 @@ def shutdown(self):
 
 # Note: This class is used by version 2
 class CortxClusterManager:
-    def __init__(self):
+    def __init__(self,default_log_enable = True):
```

Comment:
Please format your code according to PEP8 standard. This is also applicable for all other files in this merge request.